src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/PageViewModel.cs

```diff
@@ -98,4 +107,17 @@ protected virtual void FetchProperty(string propertyName)
     }
 
     protected void UpdateProperty(string propertyName) => Task.Factory.StartNew(() => { OnPropertyChanged(propertyName); }, CancellationToken.None, TaskCreationOptions.None, Scheduler);
+
+    protected void ShowException(Exception ex)
+    {
+        Task.Factory.StartNew(
+            () =>
+        {
+            ErrorMessage = $"{ex.Message}\n{ex.Source}\n{ex.StackTrace}\n\nThis is due to a bug in the extension's code.";
```

Comment:
Should we have this be a note about reaching out to the extension author? Can we grab a url or some info about the extension contact to bubble up here for the user?

Reply:
That's a great idea - something i'll stick on the backlog